admin/metabox/class-metabox-multiple-keywords-config.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Admin\Metabox
+ */
+
+/**
+ * Button to show the premium upsell.
+ */
+class WPSEO_Metabox_Multiple_Keywords_Config {
+
+	/**
+	 * Enqueues the translations necessary for the multiple keywords modal + button
+	 *
+	 * @return void
+	 */
+	public function enqueue_translations() {
+
+		$multiple_keywords_modal_config = array(
+			'openButtonIcon' => '',
+			'intl'           => array(
+				'open'           => __( '+ Add additional keyword', 'wordpress-seo' ),
```

Comment:
Not sure if it's a good idea to have the `+` symbol hard-coded in the translation string.